src/content/docs/infrastructure/host-integrations/host-integrations-list/github-integration.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Github integration
+tags:
+  - New Relic integrations
+  - Github integration
+metaDescription: Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
+freshnessValidatedDate: 2024-10-29
+---
+
+
+Set up automatic ingestion of your GitHub Dependabot events by using webhook for ongoing ingestion and historical import for past data.
```

Comment:
Can we please make the problem statement clearer in the description? What problem does this integration solve and what's in it for the users should come through clearly. For example, does the following work?

The title could be: Enhance security monitoring by integrating GitHub Dependabot with New Relic

The description could be:

Streamline vulnerability management and dependency tracking by automatically ingesting GitHub Dependabot events into New Relic. To ingest real-time events to New Relic, set up a webhook. To import past data, set up a historical import mechanism. This integration helps you optimize dependency management and reduce risk.

Reply:
Updated!